recipes/python-syringe-pump/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.2.1" %}
+{% set name = "python-syringe-pump" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Ddedalus/syringe-pump/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1b89eeebd0ad89a2e54c3f8af2e75087d6da6299dbe885df786594b4903c5758
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.10
+    - poetry-core
+
+  run:
+    - python >=3.10
+    - pydantic >=2.0.0
+    - aioserial >=1.3.1
```

Comment:
1.3.1 hasn't be published on conda-forge yet. I think it would be fine to put 1.3.0 here.
Or wait for https://github.com/conda-forge/aioserial-feedstock/pull/7 to be merged.

Reply:
Version 1.3.1 is now available!